packages/guides/src/Compiler/NodeTransformers/TocNodeWithDocumentEntryTransformer.php

```diff
@@ -45,7 +52,7 @@ public function leaveNode(Node $node, CompilerContext $compilerContext): Node|nu
 
         foreach ($files as $file) {
             foreach ($documentEntries as $documentEntry) {
-                if (!$this->isEqualAbsolutePath($documentEntry, $file, $currentPath, $glob) && !$this->isEqualRelativePath($documentEntry, $file, $currentPath, $glob)) {
+                if (!$this->isEqualAbsolutePath($documentEntry->getFile(), $file, $currentPath, $glob) && !$this->isEqualRelativePath($documentEntry->getFile(), $file, $currentPath, $glob)) {
```

Comment:
isEqualAbsolutePath is static so is isEqualRelativePath. Calls should be `self::` not using `$this`